agent/internal/containers.go

```diff
@@ -222,6 +242,45 @@ func gpuDeviceRequests(cont cproto.Container) []dcontainer.DeviceRequest {
 	}
 }
 
+func injectRocmDeviceRequests(cont cproto.Container, hostConfig *dcontainer.HostConfig) error {
+	uuids := cont.DeviceUUIDsByType(device.ROCM)
+
+	if len(uuids) == 0 {
+		return errors.New("no rocm device uuids")
+	}
+
+	hostConfig.SecurityOpt = append(
+		hostConfig.SecurityOpt, "seccomp=unconfined")
+	hostConfig.GroupAdd = append(hostConfig.GroupAdd, "video")
```

Comment:
why do we need to do this? can you leave comments or point me to docs so i can understand?